### PR TITLE
OCP4: Update policy.json verification

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

The new policy.json file expands to:
{
  "default": [{"type": "reject"}],
  "transports": {
    "docker": {
      "registry.access.redhat.com": [
        {
          "type": "signedBy",
          "keyType": "GPGKeys",
          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
        }
      ],
      "registry.redhat.io": [
        {
          "type": "signedBy",
          "keyType": "GPGKeys",
          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
        }
      ],
      "image-registry.openshift-image-registry.svc:5000": [{"type": "insecureAcceptAnything"}],
      "quay.io/complianceascode": [{"type": "insecureAcceptAnything"}],
      "quay.io/compliance-operator": [{"type": "insecureAcceptAnything"}],
      "quay.io/keycloak": [{"type": "insecureAcceptAnything"}],
      "quay.io/openshift-release-dev": [{"type": "insecureAcceptAnything"}],
      "ghcr.io/complianceascode": [{"type": "insecureAcceptAnything"}],
      "registry.build02.ci.openshift.org": [{"type": "insecureAcceptAnything"}],
      "registry.build01.ci.openshift.org": [{"type": "insecureAcceptAnything"}],
      "registry.build03.ci.openshift.org": [{"type": "insecureAcceptAnything"}],
      "registry.build04.ci.openshift.org": [{"type": "insecureAcceptAnything"}]
      "registry.build05.ci.openshift.org": [{"type": "insecureAcceptAnything"}]
      "registry.build06.ci.openshift.org": [{"type": "insecureAcceptAnything"}]
      "registry.build07.ci.openshift.org": [{"type": "insecureAcceptAnything"}]
      "registry.build08.ci.openshift.org": [{"type": "insecureAcceptAnything"}]
      "registry.build09.ci.openshift.org": [{"type": "insecureAcceptAnything"}]
    }
  }
}

In other words, we add several more registry.buildXY.ci.openshift.org
entries up to 9 and also ghcr.io/complianceascode.

We could and should add ghcr.io/complianceascode as a signed registry,
but let's do that in a separate patch to unblock CI.

#### Rationale:

Making OCP CI more stable

### CODE REVIEW

**Comment by jhrozek** [comment]
@Vincent056 turns our you were right!
can you review this PR, please?

**Comment by jhrozek** [comment]
/retest
cluster install error already..

**Comment by jhrozek** [comment]
/retest
cluster installation is really foobar today

**Comment by jhrozek** [comment]
wtf, now the operator is not installing?!?
/retest

**Comment by jhrozek** [comment]
I updated the list of registries based on what's really in CI now.
@Vincent056 PTAL again

**Review by Vincent056 — APPROVED** [review]

**Comment by Vincent056** [comment]
thanks for the PR!
